panels/templates/panel_app_form.html

```diff
@@ -9,6 +9,9 @@
             isNotAnHour = $.val('length') != {{ c.SIXTY_MIN }};
         setVisible($('#other-length'), isOther);
         $.field('length_text').prop('required', isOther);
+        if (!isOther) {
```

Comment:
Yeah, I agree that we should blank out the field if the user goes from Other to another option. This is totally fine to do that as long as we're sure West and Labs' data are good. Worst case, we have it all in the tracking table, too.